mix.exs

```diff
@@ -22,7 +22,7 @@ defmodule VerkWeb.Mixfile do
 
   def application do
     [mod: {VerkWeb, []},
-     env: [{VerkWeb.Endpoint, [http: [port: 4000], cache_static_manifest: "priv/static/manifest.json", server: true]}],
+     env: [{VerkWeb.Endpoint, [http: [port: 4000], server: false]}],
```

Comment:
I would keep the cache_static_manifest cause when I release the verk_web package I generate the manifest. 
`mix do phoenix.digest, hex.publish`

This would also mean that we would not need to push the static assets to the repository. The problem is that as a git dependency it wouldn't work out of the box I think so maybe we can do both: put static files inside the repo and cache the manifest?